includes/discovery/sensors/dbm/fabos.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$multiplier = 1;
+$divisor    = 1;
+
+foreach ($pre_cache['fabos_Sfp'] as $index => $entry) {
+    $entPhysicalIndex = substr($index, -2, 2);
+    $entPhysicalIndex = trim($entPhysicalIndex, '.');
+    $entPhysicalIndex          = $entPhysicalIndex + 1073741823;
+
+    if (is_numeric($entry['swSfpRxPower']) && $entry['swSfpRxPower'] != -99999999) {
+        $oid                       = '.1.3.6.1.4.1.1588.2.1.1.1.28.1.1.4.' . $index;
+        $current                   = $entry['swSfpRxPower'];
+        $limit_low                 = 3;
+        $limit                     = -5;
+        $entPhysicalIndex_measured = 'ports';
+        $dbquery                   = dbFetchRows("SELECT `ifDescr` FROM `ports` WHERE `ifIndex`= ? AND `device_id` = ? AND `ifAdminStatus` = 'up'", [$entPhysicalIndex, $device['device_id']]);
```

Comment:
This query is why the sensors are showing up in the test data.  It causes the sensors not to be discovered until after the poller has run.

Reply:
Any suggestion what to change? I "borrowed" the code from other dbm sensors so this is how (most) other dbm sensors are working but maybe other types of sensors are programmed in another way.

Reply:
Fixing that is probably outside the scope of this PR...